### PR TITLE
add new term MONDO:0100125 hallucinogen-persisting perception disorder

### LINKED ISSUE #1508: Hallucinogen-Persisting Perception Disorder
Hallucinogen-Persisting Perception Disorder

According to DSM-5, Hallucinogen Persisting Perception Disorder is caused by intoxication with hallucinogen drugs, especially LSD.  It is defined as the recurrence of perceptive disturbances that firstly develop during intoxication. The contents of the perception and visual imagery range extensively. DSM-5 and previous DSM editions report a list of the most common symptoms experienced by HPPD patients, but only a few symptoms have been described in the professional literature. The main group of symptoms reported by Criterion A of the DSM-5 are visual disturbances. In fact, as in the vast majority of induced psychoses, visual hallucinations are notably more common than auditory (PMID: 29547576)
PMID: 27822679 
ICD-10-CM code: F16.183
MSD-5 code 292.89  F16.983
Parent term (Substance use related  disorders  -   Hallucinogen abuse]

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #1508

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
I wasn't totally sure how to reference the DSM code: DSM-5:292.89(F16.983)

**Comment by nicolevasilevsky** [comment]
here is a website that describes that code: https://www.theravive.com/therapedia/hallucinogen-persisting-perception-disorder-dsm--5-292.89-(f16.983)

**Comment by galyea-gard** [comment]
Hi!  I am not able to find the disorder in the MONDO ontology (?).  I looked under perceptual disorders and substance abuse.  Of note, the DSM considers it under substance abuse, this is because the medical history relates this condition to previous substance use.  Thank you

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -432964,6 +432964,16 @@ intersection_of: MONDO:0005550 ! infectious disease
 intersection_of: realized_in_response_to_stimulus NCBITaxon:1335626 ! Middle East respiratory syndrome-related coronavirus
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100125
+name: hallucinogen-persisting perception disorder
+def: "A perceptual disorder caused by intoxication with hallucinogen drugs, especially LSD. It is characterized by the recurrence of perceptive disturbances that first develop during intoxication. The contents of the perception and visual imagery range extensively and symptoms may include visual disturbances, hallucinations, and psychoses." [PMID:29547576]
+synonym: "flashbacks" EXACT [DSM-5:292.89(F16.983), ICD10:F16.183]
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:432971]: I'm not sure this should be an exact synonym

It seems this term is used for any involuntary memory 
https://en.wikipedia.org/wiki/Flashback_(psychology)#cite_note-ball2006-3

Perhaps "LSD flashback"

But this seems to name the phenomena/phenotype, not the disorder.

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:432971]: I agree, LSD flashback seems more like the phenomena. "Flashbacks" was a synonym in DSM-V and ICD10, but I can remove it.

**Pull request merged**